grafana-formula/grafana/files/containers/grafana.container

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Grafana Container
+
+[Container]
+Label=app=grafana
+ContainerName=grafana
+Image=registry.opensuse.org/devel/bci/sle-15-sp6/containerfile/suse/grafana:9.5.8
```

Comment:
The same as  the other images, it needs to be configurable and for suse manager it needs to come from suse.registry.com with proper authentication

Reply:
ack